modules/ROOT/pages/api-led-test.adoc

```diff
@@ -32,27 +32,26 @@ image:configure-validate.png[Validate size configuration tab]
   .. *Max*: `30`
   .. *Error options > Message*: `The specified message is either too long or too short.`
 . From the Studio main menu, select *File > Save All* to save your changes.
-. Right-click in the canvas and select *Run project hello-world*.
-. After you see the console message indicating `hello-world` is deployed, send a request using Advanced Rest Client or a similar tool:
+. Right-click in the canvas and select *Run project hello-world-app*.
+. After you see the console message indicating `hello-world-app` is deployed, send a request using Advanced Rest Client or a similar tool:
 +
 [source, xml]
 ----
 http://localhost:8081/api/greeting
 ----
 +
-Your request client should report response headers containing `200 OK` and `Today the greeting is Hello.`. This demonstrates the expected behavior when the greeting is within the limits you set in the Validate size operation.
+Your request client reports response headers containing `200 OK` and `Today the greeting is Hello.`. This demonstrates the expected behavior when the greeting is within the limits you set in the Validate size operation.
 
-. Right-click the canvas and select *Stop project hello-world*.
+. Right-click the canvas and select *Stop project hello-world-app*.
 
 To test that the validation works when the payload is too short, use the debugger to step through each event before the error is triggered.
 
-. In the `hello-world` project in Studio, open `hello-world.xml`.
+. In the `hello-world-app` project in Studio, open `hello-world-api.xml`.
 . Click *Set Payload*, and then change the value to `Hi.`.
-. In *Mule Palette*, select *Favorites > Logger*, and then drag and drop the logger in between *Set Payload* and *Validate size*. The logger will help stop the process to help debugging. No configuration of the logger is needed.
+. In *Mule Palette*, select *Favorites > Logger*, and then drag the logger in between *Set Payload* and *Validate size*. The logger stops the process to help debugging. No configuration of the logger is needed.
```

Comment:
```suggestion
. In *Mule Palette*, select *Favorites > Logger*, and then drop the logger between *Set Payload* and *Validate size*. The logger stops the process to help debugging. No configuration of the logger is needed.
```

Reply:
Style guide says just use drag. And that's the action here--dragging it over.